build-system/pr-check.js

```diff
@@ -164,36 +196,76 @@ function determineBuildTargets(filePaths) {
 
 const command = {
   testBuildSystem: function() {
+    const startTime = startTimer('testBuildSystem');
     execOrDie('npm run ava');
+    stopTimer('testBuildSystem', startTime);
   },
   buildRuntime: function() {
+    let startTime = startTimer('buildRuntime: gulp clean');
     execOrDie(`${gulp} clean`);
+    stopTimer('buildRuntime: gulp clean', startTime);
+
+    startTime = startTimer('buildRuntime: gulp lint');
```

Comment:
instead of doing these for each execOrDie invocation what do you think about just embedding the stopTimer call in execOrDie?